components/local-app/cmd/workspace-create.go

```diff
@@ -143,10 +143,53 @@ var workspaceCreateOpts struct {
 	Editor         string
 }
 
+func classCompletionFunc(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
@mustard-mh could we decouple this from here and move into the `helpers` package 🙏?

Reply:
@filiptronicek  completion (api call + simple fmt.Sprintf) will only be used in cmd, if we move them to `helpers`, we need to move `getGitpodClient` and `rootTestingOpts (using in unit test)` out too. Not that worth to do it.

We can call `classCompletionFunc` any where in cmd package, so it's reusable now (i.e. `workspace up` cmd)

Reply:
I think we could follow the approach of `OpenWorkspaceInPreferredEditor`, where we pass in the client (easier for testing with client mocks). Feel like it's good to avoid coupling commands together through functions.

I think like this it's also alright. Leaving it up to ya